js/index.js

```diff
@@ -20,4 +20,58 @@ for(let i = 0; i < skills.length; i++){
 	let li = document.createElement('li');
 	li.innerHTML = skills[i];
 	skillsList.appendChild(li);
-}
\ No newline at end of file
+}
+
+
+let messageForm = document.querySelector('[name="leave_message"]');
+messageForm.addEventListener("submit", (e) => {
+	
+	
+	//can't use the function declaration way!
+	//MUST use arrow syntax.
```

Comment:
I think you actually could use a standard function declaration here, actually! The only time where arrow functions are necessary / helpful, are when you are attempting to access the `this` context object of the enclosing scope, which you aren't doing here.